### ISSUE TITLE
Branch 1.0 renders json examples as "[object Object]"

### ISSUE THREAD

**Issue by yasenslavov** [raml2html/raml2html#233]
Using NodeJS and npm versions:
```
node -v
v5.10.1
npm -v
3.8.3
```

In the following raml v1.0 definition:
```
/token:
  post:
    queryParameters:
      user_id:
        type: integer
    responses:
      200:
        body:
          application/json:
            properties:
              access_token:
                type: string
              token_type:
                type: string
              expires_at:
                type: string
                description: Datetime in the format 'YYYY-MM-DD HH:MM:SS' in UTC timezone, expiry date of the token.
            example: |
              {
                "access_token": "some token",
                "token_type": "bearer",
                "expires_at": "2016-07-28 12:07:14"
              }
```

The example node gets rendered as:
![screen shot 2016-07-21 at 15 41 28](https://cloud.githubusercontent.com/assets/19605805/17022956/8c3a332a-4f59-11e6-809c-46ff33cee588.png)

**Comment by sichvoge** [comment]
I think that's related to https://github.com/raml2html/raml2html/pull/210

**Comment by tr3ddy** [comment]
Should be fixed with #239
